### ISSUE TITLE
add symbols to the repl

### ISSUE THREAD

**Issue by lunixbochs** [lunixbochs/usercorn#293]

**Comment by Nokel81** [comment]
If this implies that the symbol names would be used during trace that would be amazing, maybe being able to link to the raw assembly files so comments are kept as well

**Comment by lunixbochs** [comment]
trace already has symbols, they're just not used in memory/register/jump target addrs

**Comment by Nokel81** [comment]
That is what I meant, that would be very nice to be able to see the addrs and the symbols

**Comment by lunixbochs** [comment]
That has nothing to do with this issue, and the trace UI already has access to symbols

**Comment by Nokel81** [comment]
I am sorry then, I didn't quite understand what this issue was about. Should I open a new issue then?

**Comment by lunixbochs** [comment]
Yes. This issue is about using symbols for stuff like breakpoints and address math when running commands in the REPL
